### DISCUSSION TITLE
How to access the base font size set

### DISCUSSION THREAD

**Discussion by narendrashetty** [twilio-labs/paste#2203] [General] [answered]
So in Flex, we're dynamically setting the maxHeight of textarea and to determine the right height we need to know what the base font size is.

Since lineHeight is represented in rem and to convert it to pixel, we need the base font size set.

Is there a way/api in Paste to find that out?

**Comment by gloriliale** [comment, marked as answer]
Hi @narendrashetty! The `maxHeight` value of our Paste TextArea component is fixed at our `size-30` token. Could you give us some more context about what you're building and why you're looking to have dynamic behavior? We might be able to find a more suitable solution!

Also, I think the [anatomy](https://paste.twilio.design/components/textarea/#anatomy) and [props](https://paste.twilio.design/components/textarea/#textarea-props) tables, coupled with our [tokens page](https://paste.twilio.design/tokens/#paste-docs-content-area), would be the best place to find details about sizing. (though `maxHeight` was not a property we listed in this component, so that's on us)
